addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/test/java/org/nuxeo/ecm/blob/s3/TestS3BlobStoreNoCaching.java

```diff
@@ -37,4 +38,9 @@ public void testFlags() {
         assertTrue(bs.getKeyStrategy().useDeDuplication());
     }
 
+    @Override
+    public void testBlobGetFile() {
+        new AssumptionViolatedException("S3BlobStore with no caching has no File");
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [RV: Exception created and dropped rather than thrown (RV_EXCEPTION_NOT_THROWN)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rv-exception-created-and-dropped-rather-than-thrown-rv-exception-not-thrown).
This code creates an exception (or error) object, but doesn't do anything with it. For example, something like
It was probably the intent of the programmer to throw the created exception:

Reply:
@aimiebell thanks. Please indicate in your message that this is a bot, it's bad form hide this info.